core/services/vrf/listener_v2.go

```diff
@@ -400,8 +400,17 @@ func (lsn *listenerV2) processPendingVRFRequests(ctx context.Context) {
 		sub, err := lsn.coordinator.GetSubscription(&bind.CallOpts{
 			Context: ctx,
 		}, subID)
+
+		if err != nil && strings.Contains(err.Error(), "execution reverted") {
+			lsn.l.Warnw("Subscription not found", "subID", subID, "err", err)
+			for _, req := range reqs {
+				processed[req.req.RequestId.String()] = struct{}{}
```

Comment:
Can you add a log message here for each of these requests, saying that we are abandoning them since the sub has been canceled? This makes it easy to see what happened if you're searching logs for a specific request ID.

Also do we need to markLogAsConsumed for each of these?

Reply:
As discussed, will call markLogAsConsumed here and then follow up on separate task for calling markLogAsConsumed for all processed requests. https://app.shortcut.com/chainlinklabs/story/39773/mark-processed-vrf-requests-as-consumed-in-log-broadcaster